tests/test_dependencies.py

```diff
@@ -1,240 +0,0 @@
-"""
-Copyright 2021 Inmanta
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-
-Contact: code@inmanta.com
-"""
-
-# Note: These tests only function when the pytest output is not modified by plugins such as pytest-sugar!
-import logging
-import os
-import tempfile
-
-import pytest
-
-import pytest_inmanta.plugin
-import utils
-from inmanta import env
-from pytest_inmanta.core import (
-    SUPPORTS_LEGACY_PROJECT_PIP_INDEX,
-    SUPPORTS_PROJECT_PIP_INDEX,
-)
-from pytest_inmanta.parameters import pip_index_url
-
-
-def test_transitive_v2_dependencies(
```

Comment:
Why are these tests no longer relevant?

Reply:
Because they verify that a v1 module can pull in a v2 module for installation. Since we no longer install v1 modules that doesn't apply anymore.